manifests/init.pp

```diff
@@ -241,6 +250,9 @@
   Boolean $cluster_enabled       = $::redis::params::cluster_enabled,
   $cluster_config_file           = $::redis::params::cluster_config_file,
   $cluster_node_timeout          = $::redis::params::cluster_node_timeout,
+  $cluster_slave_validity_factor = $::redis::params::cluster_slave_validity_factor,
```

Comment:
can you please add datatypes to all new parameters and align the `=`?

Reply:
Done